### PR TITLE
rec: Backport 13021 to rec-4.8.x: fix setting of policy tags

### PR DESCRIPTION
**Pull request by omoerbeek**
Backport of #13021

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [X] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [X]  checked that this code was merged to master

### CODE REVIEW

**Comment by omoerbeek** [comment]
Looking into the regress failures

File: regression-tests.recursor-dnssec/test_Protobuf.py [hunk 1]
```diff
@@ -232,17 +232,17 @@ def checkProtobufPolicy(self, msg, policyType, reason, trigger, hit, kind):
         self.assertEqual(msg.response.appliedPolicyKind, kind)
 
     def checkProtobufTags(self, msg, tags):
-        print(tags)
```
Comment (Habbie) [inline regression-tests.recursor-dnssec/test_Protobuf.py:235]: are these changes intentional?

**Review by rgacogne — APPROVED** [review]

File: regression-tests.recursor-dnssec/test_Protobuf.py (hunk 1 above)
Comment (Habbie) [reply regression-tests.recursor-dnssec/test_Protobuf.py:235]: ok!
